templates/aem-tools/import-backup.sh.epp

```diff
@@ -68,4 +68,15 @@ FACTER_aem_username="${aem_username}" \
   --hiera_config conf/hiera.yaml \
   --execute "include aem_curator::action_import_backup"
 
+  echo "Testing if package is imported using InSpec..."
+   cd "/opt/shinesolutions/aem-aws-stack-provisioner/modules/aem_curator/inspec/profiles/inspec-aem"
```

Comment:
All shellscripts shouldn't know about provisioner. This needs to be moved to `*/aem-tools/inspec-aem` .